### PR TITLE
Feature/xivy 12667 UI rework

### PR DESCRIPTION
**Pull request by ivy-fhe**
Implemented feedback and fixed git history

### CODE REVIEW

**Review by ivy-rew — APPROVED** [review]
looks nice; ready to merge as soon as the build is green ;-)

File: threema-connector-webtest/src_test/threema/connector/webtest/MessageSingleRecipientTest.java [hunk 1]
```diff
@@ -7,21 +7,33 @@
...
+  }
+
+
   @Test
-  @Disabled
-  public void sendMessage() {
-    open(EngineUrl.createProcessUrl("threema-connector-demo/18B22F69680901D3/start.ivp"));
+  public void sendMessage(WebAppFixture fixture) {
+    String mockUrl = ThreemaServiceMock.URI.replaceAll("\\{", "%7B").replaceAll("\\}", "%7D").replaceAll("/", "%2F");
+    fixture.config("RestClients.ThreemaGateway.Url", mockUrl);
```
Comment (ivy-rew) [inline threema-connector-webtest/src_test/threema/connector/webtest/MessageSingleRecipientTest.java:37]: I don't see anything obviously wrong in theses tests; can you reproduce that stuff locally by running `mvn clean verify` in the threema repo?

Comment (ivy-fhe) [reply threema-connector-webtest/src_test/threema/connector/webtest/MessageSingleRecipientTest.java:37]: Running it just as `mvn clean verify` fails with:
```
[ERROR]   GetReceiverInfoTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
[ERROR]   MessageEncryptionTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
[ERROR]   MultipleRecipientsTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
[ERROR]   SendMessageTest » ExceptionInInitializer
[ERROR]   SingleRecipientTest » NoClassDefFound Could not initialize class ch.ivyteam.ivy.server.restricted.EngineMode
```

Comment (ivy-rew) [reply threema-connector-webtest/src_test/threema/connector/webtest/MessageSingleRecipientTest.java:37]: the normal clean verify may fails due to cached ivy engines in your local maven repo: you can purge them like this `rm -rf $HOME/.m2/repository/.cache/ivy`

**Review by ivy-rew — APPROVED** [review]
go for it

**Pull request merged**